articles/other/other-ref-pricing-tool-bugs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Known bugs with the UKCloud Pricing Estimator Tool | UKCloud Ltd
+description: Known bugs with the UKCloud Pricing Estimator Tool
+services: other
+author: Dan Baker
+
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Known bugs with the UKCloud Pricing Estimator Tool
+toc_fullpath: Reference/other-ref-pricing-tool-bugs.md
+toc_mdlink: other-ref-pricing-tool-bugs.md
+---
+
+# Known bugs in the Pricing Estimator Tool
+
+This is an accompanying article to the [Pricing Estimator Tool guide](other-how-use-pricing-tool.md). This article intends to be a constantly updated list of issues and bugs users experience will the tool. As we update the tool, this article will be updated accordingly.
+
+## Current version
+
+The current version of the tool:
+
+- **Windows version**: 10.2.8
+
+- **Mac version**: 10.2.8
+
+## Known issues
+
+### Windows
+
+1. Within the UKCloud for VMware section, a user can add additional storage GiB without selecting the Storage type – with no error message.
```

Comment:
How about:

In the UKCloud for VMware section, entering an amount of additional storage GiB without selecting the Storage type does not display an error message.